roadmap/implementers-guide/src/node/disputes/votesdb.md

```diff
@@ -0,0 +1,137 @@
+# Votes DB
+
+Responsible module for tracking
+all kinds of votes on candidates
+by all validators for a fixed number
+of sessions.
+
+
+## Remarks
+
+Votes must be persisted. Disputes might happen long after
+there was a vote, as such just keeping things in memory
+is thus insufficient, as every blip (OOM, bug, maintanance)
+could cause the node to forget the state.
+
+## ToDo
+
+* Missing Definition of Governance Mode.
+* Is a `CommittedCandidateReceipt` enough or should the whole `PoV` be included? There was talk about detachable validator signatures.
```

Comment:
`PoV` should not be included. We only store `PoV`s as erasure-coded to save on space